--- st/config.def.h	2026-02-27 16:24:22.994097989 +0100
+++ /home/nix/.local/src/st/config.def.h	2026-02-20 19:01:42.754457379 +0100
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "FiraCode Nerd Font:size=12:style=SemiBold";
-static int borderpx = 5;
+static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static int borderpx = 2;
 
 /*
  * What program is execed by st depends of these precedence rules:
@@ -34,8 +34,7 @@
  *
  * More advanced example: L" `'\"()[]{}"
  */
-// wchar_t *worddelimiters = L" ";
-wchar_t *worddelimiters =  L" `'\"()[]{}";
+wchar_t *worddelimiters = L" ";
 
 /* selection timeouts (in milliseconds) */
 static unsigned int doubleclicktimeout = 300;
@@ -75,7 +74,7 @@
 static int bellvolume = 0;
 
 /* default TERM value */
-char *termname = "xterm-256color";
+char *termname = "st-256color";
 
 /*
  * spaces per tab
@@ -96,50 +95,46 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-
-  /* 8 normal colors */
-  [0] = "#0a0f14", /* black   */
-  [1] = "#c33027", /* red     */
-  [2] = "#26a98b", /* green   */
-  [3] = "#edb54b", /* yellow  */
-  [4] = "#0ce8f0", /* blue    */
-  [5] = "#8660a5", /* magenta */
-  [6] = "#33859d", /* cyan    */
-  [7] = "#98d1ce", /* white   */
-
-  /* 8 bright colors */
-  [8]  = "#10151b", /* black   */
-  [9]  = "#d26939", /* red     */
-  [10] = "#081f2d", /* green   */
-  [11] = "#cc8615", /* yellow  */
-  [12] = "#093748", /* blue    */
-  [13] = "#9653c1", /* magenta */
-  [14] = "#599caa", /* cyan    */
-  [15] = "#d3ebe9", /* white   */
-
-  /* special colors */
-  [256] = "#0a0f14", /* background */
-  [257] = "#d7e9ea", /* foreground */
+	/* 8 normal colors */
+	"black",
+	"red3",
+	"green3",
+	"yellow3",
+	"blue2",
+	"magenta3",
+	"cyan3",
+	"gray90",
+
+	/* 8 bright colors */
+	"gray50",
+	"red",
+	"green",
+	"yellow",
+	"#5c5cff",
+	"magenta",
+	"cyan",
+	"white",
+
+	[255] = 0,
+
+	/* more colors can be added after 255 to use with DefaultXX */
+	"#cccccc",
+	"#555555",
+	"gray90", /* default foreground colour */
+	"black", /* default background colour */
 };
 
+
 /*
  * Default colors (colorname index)
- * foreground, background, cursor
+ * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 257;
-unsigned int defaultbg = 256;
-unsigned int defaultcs = 257;
+unsigned int defaultfg = 258;
+unsigned int defaultbg = 259;
+unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
 
 /*
- * Colors used, when the specific fg == defaultfg. So in reverse mode this
- * will reverse too. Another logic would only make the simple feature too
- * complex.
- */
-static unsigned int defaultitalic = 7;
-static unsigned int defaultunderline = 7;
-
-/*
  * Default shape of cursor
  * 2: Block ("█")
  * 4: Underline ("_")
